frontend/src/metabase/dashboard/components/add-card-sidebar/AddCardSidebar/AddCardSidebar.unit.spec.tsx

```diff
@@ -118,151 +121,276 @@ describe("AddCardSideBar", () => {
     expect(screen.getByText("Nothing here")).toBeInTheDocument();
   });
 
-  it("should hide all personal collections when adding questions to dashboards in the root collection (public collection)", async () => {
-    const dashboardInPublicCollection = createMockDashboard({
+  describe("dashboards in the root collection (public collection)", () => {
```

Comment:
The test suite has been restructured so it's similar to the new tests introduced in `frontend/src/metabase/containers/AddToDashSelectDashModal/AddToDashSelectDashModal.unit.spec.tsx` as I feel that is more organized and easier to understand.